Dockerfile

```diff
@@ -6,7 +6,7 @@ WORKDIR /home/builder/rust-ceramic
 # Use the same ids as the parent docker image by default
 ARG UID=1001
 ARG GID=1001
```

Comment:
I'll go remove the UID and GID bits from the parent docker container so we just use root. Github actions do not like custom users :(

Reply:
removed!